### DISCUSSION TITLE
readfactor used by PKC ?

### DISCUSSION THREAD

**Discussion by PromotheusParis** [croneter/PlexKodiConnect#1905] [General]
Hi

I wonder if the  "readfactor" parameter from Advancedsettings.xml is used by PKC.

I ask this question because When I change readfactor, I didn't see the impact on the bandwith.

An advancedsettings.xml example with  readfactor :
`<advancedsettings>
<cache>
<buffermode>1</buffermode>
<memorysize>139460608</memorysize>
<readfactor>6</readfactor>
</cache>
</advancedsettings>`

Thank you for help

**Comment by croneter** [comment]
No it's not used by PKC. There's no reason for PKC to use it - PKC tells Kodi which File to play, then everything else is entirely up to Kodi. So if you have stuttering playback or other issues, PlexKodiConnect cannot be the culprit 😉

**Comment by PromotheusParis** [comment]
Ok thank you for your reply
